Index: tix-8.4.3/PyTix-2.0/2.0/_tkinter.c
===================================================================
diff -urNp -x '*.orig' Tix8.4.3.org/PyTix-2.0/2.0/_tkinter.c Tix8.4.3/PyTix-2.0/2.0/_tkinter.c
--- Tix8.4.3.org/PyTix-2.0/2.0/_tkinter.c	2000-11-22 08:59:38.000000000 +0100
+++ Tix8.4.3/PyTix-2.0/2.0/_tkinter.c	2023-07-23 13:34:43.723720858 +0200
@@ -41,6 +41,7 @@ Copyright (C) 1994 Steen Lumholt.
 #define MAC_TCL
 #endif
 
+#define USE_INTERP_RESULT 1
 #include <tcl.h>
 #include <tk.h>
 #ifdef WITH_TIX
diff -urNp -x '*.orig' Tix8.4.3.org/PyTix-2.0/2.0/tkappinit.c Tix8.4.3/PyTix-2.0/2.0/tkappinit.c
--- Tix8.4.3.org/PyTix-2.0/2.0/tkappinit.c	2000-11-22 08:59:51.000000000 +0100
+++ Tix8.4.3/PyTix-2.0/2.0/tkappinit.c	2023-07-23 13:34:43.723720858 +0200
@@ -12,6 +12,7 @@
    it explicitly, e.g. tkapp.eval("load {} Blt").
  */
 
+#define USE_INTERP_RESULT 1
 #include <tcl.h>
 #include <tk.h>
 
diff -urNp -x '*.orig' Tix8.4.3.org/generic/tix.h Tix8.4.3/generic/tix.h
--- Tix8.4.3.org/generic/tix.h	2008-02-28 05:35:16.000000000 +0100
+++ Tix8.4.3/generic/tix.h	2023-07-23 13:34:43.723720858 +0200
@@ -27,6 +27,7 @@ extern "C" {
 #endif
 
 #ifndef _TK
+#define USE_INTERP_RESULT 1
 #include <tk.h>
 #endif
 
diff -urNp -x '*.orig' Tix8.4.3.org/generic/tixPort.h Tix8.4.3/generic/tixPort.h
--- Tix8.4.3.org/generic/tixPort.h	2005-03-25 21:15:53.000000000 +0100
+++ Tix8.4.3/generic/tixPort.h	2023-07-23 13:34:43.723720858 +0200
@@ -19,6 +19,7 @@
 #define _TIX_PORT_H_
 
 #ifndef _TCL
+#define USE_INTERP_RESULT 1
 #include "tcl.h"
 #endif
 
diff -urNp -x '*.orig' Tix8.4.3.org/generic/tixUtils.c Tix8.4.3/generic/tixUtils.c
--- Tix8.4.3.org/generic/tixUtils.c	2008-02-28 05:29:17.000000000 +0100
+++ Tix8.4.3/generic/tixUtils.c	2023-07-23 13:34:43.723720858 +0200
@@ -13,6 +13,7 @@
  * $Id: tixUtils.c,v 1.13 2008/02/28 04:29:17 hobbs Exp $
  */
 
+#define USE_INTERP_RESULT 1
 #include <tcl.h>
 #include <tixPort.h>
 #include <tixInt.h>
