diff -Nru tix-8.1.4.orig/unix/tk8.3/Makefile.in tix-8.1.4/unix/tk8.3/Makefile.in
--- tix-8.1.4.orig/unix/tk8.3/Makefile.in	Wed Dec 11 08:18:41 2002
+++ tix-8.1.4/unix/tk8.3/Makefile.in	Wed Feb 12 13:41:02 2003
@@ -90,11 +90,7 @@
 #
 LIBS_TCLONLY = @TCL_BUILD_LIB_SPEC@ @TCL_LIBS@
 
-RUN_TCLSH	= TCL_LIBRARY=$(TCL_SRC_DIR)/library \
-		  TK_LIBRARY=$(TK_SRC_DIR)/library \
-		  LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:.:$(TCL_SRC_DIR)/unix \
-	          $(TCL_BIN_DIR)/tclsh
-
+RUN_TCLSH	= /usr/bin/tclsh
 
 #----------------------------------------------------------------
 # The information below is modified by the configure script when
